### DISCUSSION TITLE
OSX builds are timing out

### DISCUSSION THREAD

**Discussion by dilawar** [community/community#26571] [Actions]
<p>Our OSX builds are timing out for last week (example <a href="https://github.com/ssandrews/Smoldyn/runs/2454286828?check_suite_focus=true" class="inline-onebox" rel="noopener nofollow ugc">Some tweaks to ubuntu build. · ssandrews/Smoldyn@50f7401 · GitHub</a>). I can’t debug and not sure what is going on. Are others are facing the same issue? Any pointers regarding debugging this would be very helpful.</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>You might try turning on some debugging to see if that helps:  <a href="https://docs.github.com/en/actions/managing-workflow-runs/enabling-debug-logging" class="inline-onebox">Enabling debug logging - GitHub Docs</a></p>
<p>But in looking at your build history, this just started a few days ago, so something must have changed in your code (I’m assuming because the workflow files seem the same). What major changes did you make? Did you add new tests that are possibly hanging things up?</p>

**Comment by dilawar** [comment]
<p>Thanks <a class="mention" href="/mickeygousset">@mickeygousset</a></p>
<p>You were right. One test was added that was blocking by default (there was no timeout set for it). Fixed it. Things should be back to normal soon.</p>
<p>I was expecting Travis like logs by default: logs are flushed when a new line is added. Seems like logs are printed at the end of action on github.</p>
